emission/tests/analysisTests/intakeTests/TestPipelineRealData.py

```diff
@@ -812,6 +812,57 @@ def testShankariUntrackedTimeJul20(self):
             self.assertLessEqual(ct["metadata"]["write_ts"], end_run)
         self.assertEqual(countUntrackedTime, 0)
 
+    def testMultiOutOfOrderAug6(self):
+        # https://github.com/e-mission/e-mission-docs/issues/1122
+        # https://github.com/e-mission/e-mission-server/pull/1040
+        dataFile = "emission/tests/data/real_examples/multi_ooo_aug_6_2024"
+        etc.setupRealExample(self, dataFile)
+        etc.runIntakePipeline(self.testUUID)
+        ts = esta.TimeSeries.get_time_series(self.testUUID)
+
+        raw_trips = ts.find_entries(["segmentation/raw_trip"], None)
+        self.assertEqual(len(raw_trips), 15)
```

Comment:
future fix: we can save the expected "ground truth" in and run the same tests as before. That will give us more unusal test cases for all stages, not just this one.